core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileClassWarehouse.scala

```diff
@@ -202,38 +204,106 @@ class SQLExecutionInfoClass(
   }
 }
 
-case class SQLAccumProfileResults(appIndex: Int, sqlID: Long, nodeID: Long,
-    nodeName: String, accumulatorId: Long, name: String, min: Long, median:Long,
-    max: Long, total: Long, metricType: String, stageIds: String) extends ProfileResult {
-  override val outputHeaders = Seq("appIndex", "sqlID", "nodeID", "nodeName", "accumulatorId",
-    "name", "min", "median", "max", "total", "metricType", "stageIds")
+case class SQLAccumProfileResults(
+    appIndex: Int,
+    sqlID: Long,
+    nodeID: Long,
+    nodeName: String,
+    accumulatorId: Long,
+    name: String,
+    min: Long,
+    median: Long,
+    max: Long,
+    total: Long,
+    metricType: String,
+    stageIds: String) extends ProfileResult {
+
+  override val outputHeaders = {
+    Seq("appIndex",
+      "sqlID",
+      "nodeID",
+      "nodeName",
+      "accumulatorId",
+      "name",
+      "min",
+      "median",
+      "max",
+      "total",
+      "metricType",
+      "stageIds")
+  }
+
   override def convertToSeq: Seq[String] = {
-    Seq(appIndex.toString, sqlID.toString, nodeID.toString, nodeName, accumulatorId.toString,
-      name, min.toString, median.toString, max.toString, total.toString, metricType, stageIds)
+    Seq(appIndex.toString,
+      sqlID.toString,
+      nodeID.toString,
+      nodeName,
+      accumulatorId.toString,
+      name,
+      min.toString,
+      median.toString,
+      max.toString,
+      total.toString,
+      metricType,
+      stageIds)
   }
+
   override def convertToCSVSeq: Seq[String] = {
-    Seq(appIndex.toString, sqlID.toString, nodeID.toString,
-      StringUtils.reformatCSVString(nodeName), accumulatorId.toString,
-      StringUtils.reformatCSVString(name), min.toString, median.toString, max.toString,
-      total.toString, StringUtils.reformatCSVString(metricType),
+    Seq(appIndex.toString,
+      sqlID.toString,
+      nodeID.toString,
+      StringUtils.reformatCSVString(nodeName),
+      accumulatorId.toString,
+      StringUtils.reformatCSVString(name),
+      min.toString,
+      median.toString,
+      max.toString,
+      total.toString,
+      StringUtils.reformatCSVString(metricType),
       StringUtils.reformatCSVString(stageIds))
   }
 }
 
-case class AccumProfileResults(appIndex: Int, stageId: Int, accMetaRef: AccumMetaRef,
-    min: Long, median: Long, max: Long, total: Long) extends ProfileResult {
-  override val outputHeaders = Seq("appIndex", "stageId", "accumulatorId", "name", "min",
-    "median", "max", "total")
+case class AccumProfileResults(
```

Comment:
This class has no changes; only formats. Right?

Reply:
Yes, that's right.